change/@ni-nimble-components-a5fd15e6-373b-4e3c-b562-fc2e440df26c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Create token representing the table's height with all rows visible; **Breaking change:** the table now specifies a max-height; if a different max-height is required, it needs to be configured.",
```

Comment:
@mollykreis this PR looks ready to merge, dunno if you are waiting on anything

Reply:
I'm planning on updating the `max-height` of SLE clients first.

Reply:
Ahh, makes sense 👍